packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -224,6 +241,49 @@ class DefaultMaterialLocalizations implements MaterialLocalizations {
   @override
   String get viewLicensesButtonLabel => _nameToValue['viewLicensesButtonLabel'];
 
+  @override
+  String get anteMeridiemAbbreviation => _nameToValue["anteMeridiemAbbreviation"];
+
+  @override
+  String get postMeridiemAbbreviation => _nameToValue["postMeridiemAbbreviation"];
+
+  /// Converts the ICU short time pattern provided for key "timeOfDayFormat"
+  /// in the [localizations] map into one of [TimeOfDayFormat] enum values.
```

Comment:
We don't export `localizations` yet, so this will be a broken link.

Reply:
Removed references to implementation details.